pkg/templates/notificationtemplates/notification_generator_test.go

```diff
@@ -0,0 +1,104 @@
+package notificationtemplates_test
+
+import (
+	"testing"
+
+	"github.com/codeready-toolchain/host-operator/pkg/templates/notificationtemplates"
+
+	"github.com/pkg/errors"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+func TestCreateOrUpdateResources(t *testing.T) {
+	logf.SetLogger(zap.Logger(true))
+	t.Run("ok", func(t *testing.T) {
+		t.Run("get notification templates", func(t *testing.T) {
+			// given
+			assets := notificationtemplates.NewAssets(notificationtemplates.AssetNames, notificationtemplates.Asset)
+
+			// when
+			template, err := notificationtemplates.GetNotificationTemplates("userprovisioned", assets)
+			// then
+			require.NoError(t, err)
+			require.NotNil(t, template)
+			assert.NotEmpty(t, template.Content)
```

Comment:
you could also assert the content (or part of it), to make sure that the asset files were correctly "dispatched" in the `Content` and `Subject` fields of the `template`.

Reply:
done as part of: 32f0c75bea7b78ec9fd48726758ade2ba1b1ab66